e2e/test/scenarios/joins/reproductions/29795-native-query-join-question-misses-metadata.cy.spec.js

```diff
@@ -0,0 +1,43 @@
+import {
+  restore,
+  visualize,
+  popover,
+  openOrdersTable,
+} from "e2e/support/helpers";
+
+describe("issue 29795", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it.skip("should allow join based on native query (metabase#29795)", () => {
+    const NATIVE_QUESTION = "native question";
+    cy.createNativeQuestion({
+      name: NATIVE_QUESTION,
+      native: { query: `Select * FROM "PUBLIC"."ORDERS"` },
+    });
+
+    openOrdersTable({ mode: "notebook" });
+
+    cy.button("Join data").click();
```

Comment:
Did you check if this works?
IIRC, we had to always click on the icon to join, but it might have changed.

Reply:
definitely I checked

Reply:
double checked - it works